### PR TITLE
Run node generate keyset

### PR DESCRIPTION
**Pull request by anegg0**
This PR is a temporary fix to provide Keyset generation content to node providers/RaaS.
The content is meant to be enriched in the next two weeks.

### CODE REVIEW

File: arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+content_type: how-to
+---
+
+import UnderConstructionPartial from '../../partials/\_under-construction-banner-partial.md';
+
+<UnderConstructionPartial />
+
+The DAC has _N_ members; the AnyTrust protocol assumes that a minimum of _H_ _DAC_ members maintain integrity. _H_ is the minimum number of trusted committee members on AnyTrust chains, configurable by the chain's owner via the `assumed-honest` parameter in the keyset. In scenarios where `K = (N + 1) - H` members of the DAC pledge to grant access to specific data, they must sign and attest they have the data for storage to be considered successful.
+
+Each DAC member gets their own set of BLS public and private keys. It's important for every member to create their own new and secure BLS keys. They should do this on their own and make sure these keys are random and only for their use. If you need help generating BLS keys, please refer to our guide on generating keys in the Arbitrum documentation: [Generating BLS Keys](https://docs.arbitrum.io/node-running/how-tos/running-a-daserver#generate-key).
```
Comment (Tristan-Wilson) [inline arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md:17]: The link doesn't seem to go to the right page

File: arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+The main blockchain (parent chain) needs to know the names and public keys of all DAC members in order to validate the integrity of data being batched and posted. A 'keyset' is a list of all DAC members' public keys. It also shows how many signatures are needed to approve a Data Availability Certificate. This design lets the chain owner modify the DAC's membership over time, and it lets DAC members change their keys if needed. See [Inside AnyTrust](/how-arbitrum-works/inside-arbitrum-nitro.md#inside-anytrust) for more information.
```
Comment (Tristan-Wilson) [inline arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md:19]:
```suggestion
The main blockchain (parent chain) needs to know the public keys of all DAC members in order to validate the integrity of data being batched and posted. A 'keyset' is a list of all DAC members' public keys. It also shows how many signatures are needed to approve a Data Availability Certificate. This design lets the chain owner modify the DAC's membership over time, and it lets DAC members change their keys if needed. See [Inside AnyTrust](/how-arbitrum-works/inside-arbitrum-nitro.md#inside-anytrust) for more information.
```

Comment (Tristan-Wilson) [reply arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md:19]: The link also didn't work for me. Maybe it's only not working in preview.

File: arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+In the following section, we will provide a detailed guide on the generation of a Keyset corresponding to your individual set of keys, as well as instructions for its subsequent configuration within the chain.
+
+### Batch poster configuration
+
+AnyTrust works with a group of Data Availability Servers, forming a committee that ensures transaction data is accessible. When setting up the Nitro Batch Poster, you need to provide specific information for each committee member. This includes their URL, BLS public key, a unique single-bit identifier (bitmask) for each member, and a parameter known as assumed-honest. As mentioned before, assumed-honest refers to the minimum number of committee members that we trust.
```
Comment (Tristan-Wilson) [inline arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md:25]:
```suggestion
AnyTrust works with a group of Data Availability Servers, forming a committee that ensures transaction data is accessible. When setting up the Nitro Batch Poster, you need to provide specific information for each committee member. This includes their URL and BLS public key for each member, and a parameter known as assumed-honest. As mentioned before, assumed-honest refers to the minimum number of committee members that we trust.
```

Comment (Tristan-Wilson) [reply arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md:25]: The user doesn't need to specify the signer bitmask any more.

File: arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+To ensure data is stored properly, a certain number of committee members need to confirm they have the data. This number is calculated as `K = (N + 1) - H`, where _N_ is the total number of committee members and _H_ is the minimum number of members assumed to be honest.
+
+Someone setting up an AnyTrust node would need to first set up the committee of Data Availability Servers, including generating their BLS keys. How to do that is described [in this guide](https://developer.arbitrum.io/das/daserver-instructions#generate-key).
```
Comment (Tristan-Wilson) [inline arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md:29]: Same link as above, doesn't work.

File: arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+Here is a sample of the `JSON` configuration, taken from a past configuration of Arbitrum Nova, which was used with to the batch poster configuration. Note that due to a quirk in a configuration library we use in Nitro, the `backends` field is an escaped JSON string with `url`, `pubkey`, and `signer-mask` fields. `pubkey` is the base64 encoded BLS public key of the committee member, and `signer-mask` should be a power of 2, starting from 2^0, 2^1, etc.
```
Comment (Tristan-Wilson) [inline arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md:31]: Remove all refs to signer-mask

File: arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
...1NsgjMC6kb/UThU71zc2t98BNBeAqqfxhfyg06R437U7YS7ZHwVEFt571ixZ05tO5gTqlstC89aqIBEhl7S5rPc25H+0TPAMvjclIn2K6lvKHvu7iUwcu1ZqerG3Aegdr9W58wd85Piy+AAKBH+4jcXc+K/udljfD5hTM6MVFvumIgB2D+QlbAJgsZmUlzf7iPd9dcNd6kNJJhIT7mvrv/NQIEr/fNhGFlG+1Xy0VRhLkKv1ahuyPe/+qbsl2uxc0r6dfOAQcZudWg==\",\"signermask\":32},{\"url\":\"https://example\",\"pubkey\":\"YBN+CWUmeRP56vhb/yLjzl9Euxv67XZ5sWgKzRVDaoQyXrp/KWLKRpN8y/Rtme3JRANM3Ze8T7HY3DrducNIQxqZl1lZ5qyCODdq8x8D51T6PDFZJ81oYCZeyObpfaQKlQkyd3PnqlvPrvdpDXaQYzNvbVIQp95V4OvyUf/VP23KAiJn+8P/wI5HCeNBSpa4BAVt1+f/w4kn2KPYpcD0ao5zfWOOie9clvxd/nmiHaCitcvQwbsulcqbv/HUFlhcLAEZtnbcBTxauzsLTWDq+gZXFaLDAajVi6yHHfg238DrjU7eGRyrTPZlVFGjfJzzdggqZdXyO4GMGFxWsWzpgLb+DWiDj95nePzGUs9oE/79Ids3J0VN9ZrTvnRl1gUHqg==\",\"signermask\":64},{\"url\":\"https://example\",\"pubkey\":\"YAR40SbOOU71LW/8aEVnLfztsU1Mq+dqzZ7/8liSsx3DLYvSFCZXXwijCxuEu4wfZQeBDiXUeFLx8qBrZrU0HQLXSBoczgElfnaKoaWbaDoo9veUZnRUHw9OI2Q9Md/X6QlYo2HH24a2KP4HXZTIXixD+FjT82g9U2mof3azkCwHZd3IyQTjdbD1dA210uJfGxWalm4gWWsaOP8xG1Nl13CcxnmZEwfWkhUs/0mHZmPzFeCB9MC8hcOKZvIZjVOQFwwnXiPphD6nSgRrPnCEqqHVPG/GyGIiUNy4EtREw0GoRw30ssLsOrDUqlY7EBoxUg1x3wycHuxIGMuuyzJKwrkEWusxa7H0xsmqySR/HsP7gkJH84WNG5xgMUE6EaewWQ==\",\"signermask\":128}]"
```
Comment (Tristan-Wilson) [inline arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md:41]: The config for this now doens't need to be escaped. See the description in this PR: https://github.com/OffchainLabs/nitro/pull/2427

File: arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
```
Comment (Tristan-Wilson) [inline arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md:56]: Update to new format, as above. No need for signer mask.

File: arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+
+```shell
+{
+...
+  "data-availability": {
+    "enable": true,
+    "rpc-aggregator": {
+			"enable": true,
+			"assumed-honest": 1,
+      "backends": "[{\"url\":\"http://localhost:9876\",\"pubkey\":\"YAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA==\",\"signermask\":1}]"
```
Comment (Tristan-Wilson) [inline arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md:106]: New format, no signer mask.

**Review by Tristan-Wilson — CHANGES_REQUESTED** [review]
Thanks for putting this together.

File: arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md (hunk 4 above)
Comment (anegg0) [reply arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md:29]: That's interesting; the link works for me.
Looks like a Vercel cache issue, I've had the pb often, lately.

File: arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md (hunk 2 above)
Comment (anegg0) [reply arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md:19]: That's weird; the link works for me. Vercel cache issue, probably.

File: arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md (hunk 8 above)
Comment (anegg0) [reply arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md]: removed, thx

File: arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md (hunk 1 above)
Comment (anegg0) [reply arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md]: It should now, pls lmk if the link it points to now is relevant.

File: arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md (hunk 5 above)
Comment (anegg0) [reply arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md]: Done, your scanning the diff is appreciated

File: arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md (hunk 7 above)
Comment (anegg0) [reply arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md]: Done, your scanning the diff is appreciated

File: arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md (hunk 6 above)
Comment (anegg0) [reply arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md]: Removed escape characters.

File: arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+
+To ensure data is stored properly, a certain number of committee members need to confirm they have the data. This number is calculated as `K = (N + 1) - H`, where `N` is the total number of committee members and `H` is the minimum number of members assumed to be honest.
+
+Someone setting up an AnyTrust node would need to first set up the committee of Data Availability Servers, including generating their BLS keys. You can learn more about this topic [in this guide explaining the process of generating BLS keys ](run-arbitrum-node/data-availability-committees/deploy-das#step-1-generate-the-bls-keypair).
+
+Here is a sample of the `JSON` configuration, taken from a past configuration of <a data-quicklook-from="arbitrum-nova">Arbitrum Nova</a>, which was used with the batch poster configuration. Note that due to a quirk in a configuration library we use in Nitro, the `backends` field is an escaped JSON string with `url`, and `pubkey`. `pubkey` is the base64 encoded BLS public key of the committee member.
```
Comment (Tristan-Wilson) [inline arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md:31]: backends is no longer an escaped json string and is just normal json now and can be formatted nicely https://github.com/OffchainLabs/nitro/pull/2427

File: arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md [hunk 10]
```diff
@@ -0,0 +1,111 @@
...
...+3qflOLxWCJjDy1hbDKjNAOHEY5LluJtbkHbqrn+J61gi9gjoUL5iPfamZzeygirSv7baz2i1NsgjMC6kb/UThU71zc2t98BNBeAqqfxhfyg06R437U7YS7ZHwVEFt571ixZ05tO5gTqlstC89aqIBEhl7S5rPc25H+0TPAMvjclIn2K6lvKHvu7iUwcu1ZqerG3Aegdr9W58wd85Piy+AAKBH+4jcXc+K/udljfD5hTM6MVFvumIgB2D+QlbAJgsZmUlzf7iPd9dcNd6kNJJhIT7mvrv/NQIEr/fNhGFlG+1Xy0VRhLkKv1ahuyPe/+qbsl2uxc0r6dfOAQcZudWg=="},{"url":"https://example","pubkey":"YBN+CWUmeRP56vhb/yLjzl9Euxv67XZ5sWgKzRVDaoQyXrp/KWLKRpN8y/Rtme3JRANM3Ze8T7HY3DrducNIQxqZl1lZ5qyCODdq8x8D51T6PDFZJ81oYCZeyObpfaQKlQkyd3PnqlvPrvdpDXaQYzNvbVIQp95V4OvyUf/VP23KAiJn+8P/wI5HCeNBSpa4BAVt1+f/w4kn2KPYpcD0ao5zfWOOie9clvxd/nmiHaCitcvQwbsulcqbv/HUFlhcLAEZtnbcBTxauzsLTWDq+gZXFaLDAajVi6yHHfg238DrjU7eGRyrTPZlVFGjfJzzdggqZdXyO4GMGFxWsWzpgLb+DWiDj95nePzGUs9oE/79Ids3J0VN9ZrTvnRl1gUHqg=="},{"url":"https://example","pubkey":"YAR40SbOOU71LW/8aEVnLfztsU1Mq+dqzZ7/8liSsx3DLYvSFCZXXwijCxuEu4wfZQeBDiXUeFLx8qBrZrU0HQLXSBoczgElfnaKoaWbaDoo9veUZnRUHw9OI2Q9Md/X6QlYo2HH24a2KP4HXZTIXixD+FjT82g9U2mof3azkCwHZd3IyQTjdbD1dA210uJfGxWalm4gWWsaOP8xG1Nl13CcxnmZEwfWkhUs/0mHZmPzFeCB9MC8hcOKZvIZjVOQFwwnXiPphD6nSgRrPnCEqqHVPG/GyGIiUNy4EtREw0GoRw30ssLsOrDUqlY7EBoxUg1x3wycHuxIGMuuyzJKwrkEWusxa7H0xsmqySR/HsP7gkJH84WNG5xgMUE6EaewWQ=="}]"
```
Comment (Tristan-Wilson) [inline arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md:41]: The backends field contents is now a json array of onbjects, not a string of json.

```
{
  "data-availability": {
    "enable": true,
    "rpc-aggregator": {
      "enable": true,
      "assumed-honest": 2,
      "backends": [
        {
          "url": "http://example",
          "pubkey": "YAbcteVnZLty5qRebeswHKhdjEMVwdou+imSfyrI+yVXHOMdLWA3Nf4DGW9tVry/mhmZqJp01TaYIsREXWdsFe1S5QCNqnddyag5yZ/5Y6GZRqx0BXmHTaxPY5kHrhvGnwxmlJVbUk1xjKRFgxxTdTk3c0AfM3JaeWYTed3avV//KGGdwHC+/Z7XPWmeXCNsGhY75YuoEAK2EwcJvAZK9de6lHEwtyBWvxcmOADxo6siacalEO+OdBL9VtHvG5FqEwbjsdnILAmTcb2YYVgqyq2joW6d/uXQ685hCWWYqC8RLQqTXoyrXEjYLjEEsMe6eRV9rRoBmj5/atB3uOYwixFv7A9YI5YiRjw2MfoB4rQnJAkhW4AJQiwWcV2+3lkJBg=="
        },
        {
          "url": "http://example",
          "pubkey": "YAg1+ZXyR48kiS0FDaoon4trnBsYW80oUy+I1hDCZCotxvNQl0AjbTPD4tkTaqsX+BnIxnEpO7ondxd2Lo0cH3usnhfdKNKTmpWbs45QD5wRw4zrvEJuLeqXxAF1plXRdACubHX/SeiEx5RpJJ5wlTJYhUtk+oRFxYWtRdxtxpdVAcavfP9wdCAsaH+Ke/GjrBkmiXVfIyJ1tMhCGxpWaem5BMKaKSzflht4OnwLTOc2kA3k2MY8X4WmXLRK80vvhArO+Eq3X0TEyRN2ELaBB6/zu9zBkRnHqSfBFbe5v7J9hcUA7nfRPsWpejrmv1HTtwpVAuhBbee1646f7uN2QRyjXIp/P1l8dgZXjPlqRxXOWjXPSOOcCh+qLe4i105oGQ=="
        },
        {
          "url": "http://example",
          "pubkey": "YAXbmOUQgLs5Kntevb/PM+D08BkxAsxA95qe8KlVfFpi3R74AAVpRugyn5eboMyCUQ0Nx4w8zv+mbuXeXimJh6mFi/UmIXFhTlVvQGh85pEsvqaltERyyz/xB+zmnL0P2g2zkqZKgr5xQHc1HWOE1s6iVK08IeRah4UtrDkZCGXESaZHpOOZKNrXYRsNIHRDfRfkTHZvPgoVCFLN4jXUVScbkeygPX7JHJEMo4VA9verlPy/pMQ6TdDwZLYhADfZNws61wLriD3L965tj+ZSkWRZRpLwD5iIwud9E16SJYWtnOHBYpG5qz5KmFgrPt0gmhAKAF/kT6LkhDvxmVmRqL+1rw/SxeV0mkqW2IZfMc77k2j2/qy2HyR38krTMrZVMg=="
        },
        {
          "url": "https://example",
          "pubkey": "YAbOg53k1qOuAvJbQIlTHmo9LeVWvQBr0wzy00CLl30Y8XVt1KG8PADbkALw2O8a9Q+6ppWd7L7By+I0zG72JwoDM5CQ4COPisn4oY9EuHNMjzthI90SiuSKCGO5p/bYgwIENoF3LCt581DBS8nXsY5XjHlcp2klznvSiR0Jqjf+LOHqL/5MA4/vIvZuKJlcYQkCIehqM2j7bWcar8GQAfQif5LzZxr2pgs9sShaFC9JwfRQt2/rmxrvGlUehY5LcwLGzxyn+sBbmTujddgYP5DAifqKPfMaRD0uCo7xkwZwaPv8yizvI0z22cm1iE72lwWnGBClZlFybKO3rN+v/fq5soRLTqaHK0P/m32i4QouI6eRMMPExwqK0Kd+stUfFg=="
        },
        {
          "url": "https://example",
          "pubkey": "YA+HK4mKT9G4rnNRX30zzXvh6XHOGJaqvvL4km5YbEJI3A23/XhRQCwUFJ3D3ITzgww0YWfDnlMjlxrDQEFfCi6wVKmo4KXVA6Ks/s690d9xrurDs4JgSAxpm8CZNPCRPg7lquq9VzEyhSB+uJNmtBEobPPxxeMOt+NV9VOFMIuR1YByhDI+6JqXQ8cGdvSUlQTO0+1BYSy/2gatVSAMHHfT+zcABZvv1kxEvEpXy1Z+wUge5WTPbNbPHy9KLe5tsAxUfDhACrEY3tror9W6uTtwP3agmRuqXUP7sSUZTAa1Rh+Mc4o8Qnij2Y5UVq7AcgiDwNKJGVN6NuL/1fcx50K2ZTVX0VTBZOBo75g7Nn72Jvqu1G9Orey7ErflXyMXXQ=="
        },
        {
          "url": "https://example",
          "pubkey": "YALC7DeOtroXqegbj9RCY9aZw0cZSSpOzx7napQrwiR4+3qflOLxWCJjDy1hbDKjNAOHEY5LluJtbkHbqrn+J61gi9gjoUL5iPfamZzeygirSv7baz2i1NsgjMC6kb/UThU71zc2t98BNBeAqqfxhfyg06R437U7YS7ZHwVEFt571ixZ05tO5gTqlstC89aqIBEhl7S5rPc25H+0TPAMvjclIn2K6lvKHvu7iUwcu1ZqerG3Aegdr9W58wd85Piy+AAKBH+4jcXc+K/udljfD5hTM6MVFvumIgB2D+QlbAJgsZmUlzf7iPd9dcNd6kNJJhIT7mvrv/NQIEr/fNhGFlG+1Xy0VRhLkKv1ahuyPe/+qbsl2uxc0r6dfOAQcZudWg=="
        },
        {
          "url": "https://example",
          "pubkey": "YBN+CWUmeRP56vhb/yLjzl9Euxv67XZ5sWgKzRVDaoQyXrp/KWLKRpN8y/Rtme3JRANM3Ze8T7HY3DrducNIQxqZl1lZ5qyCODdq8x8D51T6PDFZJ81oYCZeyObpfaQKlQkyd3PnqlvPrvdpDXaQYzNvbVIQp95V4OvyUf/VP23KAiJn+8P/wI5HCeNBSpa4BAVt1+f/w4kn2KPYpcD0ao5zfWOOie9clvxd/nmiHaCitcvQwbsulcqbv/HUFlhcLAEZtnbcBTxauzsLTWDq+gZXFaLDAajVi6yHHfg238DrjU7eGRyrTPZlVFGjfJzzdggqZdXyO4GMGFxWsWzpgLb+DWiDj95nePzGUs9oE/79Ids3J0VN9ZrTvnRl1gUHqg=="
        },
        {
          "url": "https://example",
          "pubkey": "YAR40SbOOU71LW/8aEVnLfztsU1Mq+dqzZ7/8liSsx3DLYvSFCZXXwijCxuEu4wfZQeBDiXUeFLx8qBrZrU0HQLXSBoczgElfnaKoaWbaDoo9veUZnRUHw9OI2Q9Md/X6QlYo2HH24a2KP4HXZTIXixD+FjT82g9U2mof3azkCwHZd3IyQTjdbD1dA210uJfGxWalm4gWWsaOP8xG1Nl13CcxnmZEwfWkhUs/0mHZmPzFeCB9MC8hcOKZvIZjVOQFwwnXiPphD6nSgRrPnCEqqHVPG/GyGIiUNy4EtREw0GoRw30ssLsOrDUqlY7EBoxUg1x3wycHuxIGMuuyzJKwrkEWusxa7H0xsmqySR/HsP7gkJH84WNG5xgMUE6EaewWQ=="
        }
      ]
    }
  }
}
```

File: arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md [hunk 11]
```diff
@@ -0,0 +1,111 @@
...
```
Comment (Tristan-Wilson) [inline arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md:56]: same comment as above about backends

File: arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md [hunk 12]
```diff
@@ -0,0 +1,111 @@
...
+
+```shell
+{
+...
+  "data-availability": {
+    "enable": true,
+    "rpc-aggregator": {
+			"enable": true,
+			"assumed-honest": 1,
+      "backends": "[{"url":"http://localhost:9876","pubkey":"YAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=="}]"
```
Comment (Tristan-Wilson) [inline arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md:106]: same comment as above about backends

**Review by Tristan-Wilson — DISMISSED** [review]

**Comment by TucksonDev** [comment]
@anegg0 , @Tristan-Wilson , I've pushed #1584 on top of this branch, since the changes added modify the old structure significantly. I've taken into account Tristan's comments and laid out the page so it is consistent with the rest of the DAC guide. Appreciate your review there! 🙏

**Review by TucksonDev — APPROVED** [review]
Approved in #1584

**Pull request merged**